lib/order.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'dishes'
+require_relative 'takeaway'
+class Order
+  attr_reader :order_list
+
+  def initialize(order_list = {})
+    @order_list = order_list
+  end
+
+  def add_to_order(add)
```

Comment:
I'm not totally sure how this function is meant to work, maybe a comment or some info in the Readme would be handy for reviewers?